Spring-Boot/Spring-Boot-Academy/src/main/java/tw/waterball/judgegirl/springboot/academy/repositories/JpaExamRepository.java

```diff
@@ -159,11 +159,6 @@ public int countAnswersInQuestion(Question.Id id, int studentId) {
         return jpaAnswerDAO.countAllByExamIdAndProblemIdAndStudentId(id.getExamId(), id.getProblemId(), studentId);
     }
 
-    @Override
-    public boolean isExaminee(int studentId, int examId) {
-        return jpaExamineeDAO.existsById_StudentIdAndId_ExamId(studentId, examId);
-    }
```

Comment:
Why did you remove this method?

Reply:
Because we only use the `findExam` method to query an `exam` entity, we can just use `exam` to ensure has the examinee participated in the exam or not.